### ISSUE TITLE
CreateLinkage Issue

### ISSUE THREAD

**Issue by phoebe460** [Taiji-pipeline/Taiji#16]
Hello again,

So I've run the pipeline for most of today (almost there!), but now I get the CreateLinkage (Disconnect Error) and the program stops entirely. Here it is:

<img width="1173" alt="Error 5" src="https://user-images.githubusercontent.com/56165012/103840409-5f325900-5046-11eb-8577-4ad524cdd728.png">

Just wondering if this is a memory associated issue and should I increase the memory at this point? Let me know. I am also still troubleshooting my other issue regarding the bigWig files, so if I can figure that one out, I will respond to my other issue. 

Thanks again!
Phoebe

**Comment by kaizhang** [comment]
It looks like the remote computing node was shut down unexpectedly. To find out the exact reason, you need to check the log file created by the workload manager. Please check the log files for the error message.

**Comment by phoebe460** [comment]
So it looks like I was able to resolve this by just running the program without distributed computing using the following command:

```
(base) [prvaldes@tscc-2-6- Taiji]$ taiji run --config config_PV.yml -n 50 +RT@ -N50
```
Here is a screenshot of the "Output Rank" step completed:

<img width="674" alt="Program 5 Start!" src="https://user-images.githubusercontent.com/56165012/103926862-600ecd80-50ce-11eb-92e8-af15911f02aa.png">

I got the results I needed with the TF's ranked by PageRank scores, so I should be good to go. I'll ask you clarification of what the results mean in a bit. 

Otherwise, it's definitely been quite the week for me getting this pipeline to work. Thank you so much Kai for all your help and patience with this!! Again, congratulations on the paper and making this great tool!  :)
